content/sensu-go/6.3/operations/manage-business-services/_index.md

```diff
@@ -0,0 +1,198 @@
+
+For example, with business service management, you can specify that failure of certain individual components will trigger an alert to an operator mailbox and create a ticket to address the next workday.
+At the same time, for failure of a component that affects availability or performance of your entire business service, your rule template can trigger an alert to your on-call staff for immediate response.
+You can also set a delay so that a component failure causes alerts only after the failure persists for a certain amount of time or after a number of polling intervals.
+
+Business service management highlights which business services are affected by the failure of an underlying system.
+
+Business service management requires two resources that work together to achieve top-down monitoring: components and rule templates.
+
+## Components
+
+Business service [components][1] are resources for defining and managing elements of a business service.
+Here is an example component definition that references the `status-threshold` rule template:
+
+{{< language-toggle >}}
+
+{{< code yml >}}
+---
+type: Component
+api_version: bsm/v1
+metadata:
+  name: postgresql-component
+  namespace: default
+  created_by: admin
+spec:
+  services:
+    - account-manager
+    - tessen
+  interval: 60
+  cron: ""
+  query:
+    - type: labelSelector
+      value: region == us-west-2 && cmpt == psql
+  rules:
+    - template: status-threshold
+      arguments:
+        status: non-zero
+        threshold: 25
+  handlers:
+    - pagerduty
+    - slack
+{{< /code >}}
+
+{{< code json >}}
+{
+  "type": "Component",
+  "api_version": "bsm/v1",
+  "metadata": {
+    "name": "postgresql-component",
+    "namespace": "default",
+    "created_by": "admin"
+  },
+  "spec": {
+    "services": [
+      "account-manager",
+      "tessen"
+    ],
+    "interval": 60,
+    "cron": "",
+    "query": [
+      {
+        "type": "labelSelector",
+        "value": "region == us-west-2 && cmpt == psql"
+      }
+    ],
+    "rules": [
+      {
+        "template": "status-threshold",
+        "arguments": {
+          "status": "non-zero",
+          "threshold": 25
+        }
+      }
+    ],
+    "handlers": [
+      "pagerduty",
+      "slack"
+    ]
+  }
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+If you delete a resource (e.g. an entity, check, or event) that is part of one or more business service components, Sensu will automatically remove the deleted resource from the components.
+
+## Rule templates
+
+In [rule templates][2], you configure conditions for creating an event with an ECMAScript 5 (JavaScript) expression.
+When you add rule templates by reference in your component definitions, Sensu evaluate events according to the expression you configure in the rule template.
+
+For example, the component listed above references the `status-threshold` rule template:
+
```

Comment:
Again, this is the same example I used in the rule templates reference, so I'm wondering if we should use a different example.